tests/app/helpers/test_template_helpers.py

```diff
@@ -157,7 +158,7 @@ def test_get_page_header_context_census_nisra(app: Flask):
             None,
         ),
         (
-            BusinessSurveyConfig(),
+            BusinessSurveyConfig(schema=QuestionnaireSchema({"survey_id": "001"})),
```

Comment:
Don't we want to keep a Business Survey Config test with no schema present?

Reply:
Added a test for not authenticated. Current one wasn't quite right since it didn't have query param when authenticated. Didn't do no schema as if you are authed, you will have schema.

Reply:
Ok, thanks. Just because there is such scenario in the code I preferred to have something in this tests.